NimbleExtension/Sources/Typealiases/Typealias.swift

```diff
@@ -0,0 +1,21 @@
+//  swiftlint:disable:this file_name
+//  Typealias.swift
+//  NimbleExtension
+//
+//  Created by Su T. Nguyen on 9/10/20.
+//  Copyright © 2020 Nimble. All rights reserved.
+//
+
+import Alamofire
+
+public typealias Headers = Alamofire.HTTPHeaders
+public typealias Parameters = Alamofire.Parameters
+
+public typealias Callback<T> = (T) -> Void
```

Comment:
Maybe for typealiases like `Callback` and `ResultCompletion` we can make those internal and re-add them in project through the project template. This is mainly because in order to use these typealiases we need to import NimbleExtension in every file we wish to use it.